Makefile

```diff
@@ -13,10 +20,14 @@ mypy:
 	mypy --ignore-missing-imports --no-strict-optional $(MODULES)
 
 notebooks:=$(wildcard notebooks/*.py)
-$(notebooks): lint mypy
-	python $@
-	callysto $@ > $(@:.py=.ipynb)
-	scripts/publish.sh $@ $(@:.py=.ipynb)
+PIP_REQS:=$(shell cat $(LOCAL_ROOT_DIR)/requirements.txt | tr "\n" " ")
+$(notebooks): clean lint mypy
+	docker exec $(CONTAINER) bash -c "$(LEO_PIP) install --upgrade $(PIP_REQS)"
+	docker exec $(CONTAINER) mkdir -p notebooks
+	docker cp $(LOCAL_ROOT_DIR)/$@ $(CONTAINER):$(CONTAINER_ROOT_DIR)/$@
```

Comment:
The entire repo directory is mounted into the container under `CONTAINER_REPO_ROOT="/$(basename ${BDCAT_NOTEBOOKS_HOME})"`. I think this line can be removed?

Reply:
Should be addressed.